crew

```diff
@@ -713,7 +721,7 @@ def prepare_package (destdir)
         puts f.lightred
         pp = p
       end
-      abort
+      abort unless ENV['CREW_CONFLICTS_ONLY_ADVISORY'] == '1'
```

Comment:
I'm not a fan of allowing overrides like this.  You can easily miss conflicts if the environment variable remains set for other packages.